tests/data/granule.umm_json

```diff
@@ -132,5 +132,24 @@
             "Date": "2024-03-02T22:12:36.000Z",
             "Type": "Update"
         }
-    ]
+    ],
+    "Platforms": [
+            "Sentinel-1A",
+            "Sentinel-1A"
+            ],
+    "OrbitDirectionTypeEnum": "D",
```

Comment:
My intent was to document the orbit _number_ here; orbit _direction_ is in the `ASCENDING_DESCENDING` attribute below.
```suggestion
    "OrbitCalculatedSpatialDomains": [
        {"OrbitNumber": 52520},
        {"OrbitNumber": 51995}
    ],
```